### PR TITLE
feat: kyverno 1.9 in operator

### PR DESCRIPTION
**Pull request by pns-nirmata**

### CODE REVIEW

**Review by atulatnirmata — APPROVED** [review]

File: charts/enterprise-kyverno-operator/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: enterprise-kyverno-operator
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
-version: v0.1.0-rc.2
+version: v0.1.0-rc.3
 appVersion: v0.1.0-rc.1
```
Comment (kumar-mallikarjuna) [inline charts/enterprise-kyverno-operator/Chart.yaml:6]: Do we need to change the `appVersion` as well?

File: charts/enterprise-kyverno-operator/values.yaml [hunk 2]
```diff
@@ -57,7 +78,7 @@ kyverno:
     # -- Image repository
     repository: ghcr.io/nirmata/kyverno
     # -- Image tag
-    tag: v1.8.5-n4kbuild.1
+    tag: v1.9.2-n4k.nirmata.1
```
Comment (kumar-mallikarjuna) [inline charts/enterprise-kyverno-operator/values.yaml:81]: Why is this different from everywhere else/`v1.9.1`?

**Review by kumar-mallikarjuna — APPROVED** [review]

File: charts/enterprise-kyverno-operator/values.yaml (hunk 2 above)
Comment (pns-nirmata) [reply charts/enterprise-kyverno-operator/values.yaml:81]: The 1.9.2 Chart PR is still not merged, but the image was ready. So, decided to use the 1.9.2 build directly, with 1.9.1 chart.

File: charts/enterprise-kyverno-operator/Chart.yaml (hunk 1 above)
Comment (pns-nirmata) [reply charts/enterprise-kyverno-operator/Chart.yaml]: Actually, the new app version build was not ready at that time. Available now. Updated.

**Pull request merged**